### PR TITLE
2513 filters vs operations

### PR DESCRIPTION
**Pull request by jenlampton**
Fixes https://github.com/backdrop/backdrop-issues/issues/2513

### CODE REVIEW

File: core/themes/seven/templates/maintenance-page.tpl.php [hunk 1]
```diff
@@ -27,9 +27,5 @@
     </main>
   </div>
 
-  <footer role="contentinfo">
```
Comment (klonos) [inline core/themes/seven/templates/maintenance-page.tpl.php]: Was this change intentional? Seems unrelated.

File: core/themes/seven/templates/maintenance-page.tpl.php [hunk 2]
```diff
@@ -27,9 +27,5 @@
...
-    <?php print $footer; ?>
-  </footer>
-
```
Comment (klonos) [inline core/themes/seven/templates/maintenance-page.tpl.php]: Was this change intentional? Seems unrelated.

Comment (jenlampton) [reply core/themes/seven/templates/maintenance-page.tpl.php]: yeah sorry that commit was on my 1.x branch for some reason. Should be gone now :)

**Comment by jenlampton** [comment]
doing that thing you need to do to get a new sandbox.....

**Comment by quicksketch** [comment]
Looks great, thanks @jenlampton and @klonos! Merged into 1.x for 1.8.0.
